Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +16,143 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
+        }
+
+        public static int FindDigit(string equation)
+        {
+           return FixedMultiplication.FindDigit(equation);
+        }
+    }
+
+    enum EquationParts{
+        Operand1,
+        Operand2,
+        Result,
+        Invalid
+
+    }
+
+    class FixedMultiplication
+    {
+        
+        public static int FindDigit(string equation)
+        {
+           Equation ParsedEquation = ParseEquation(equation);

```

Comment:
var
it should be 'parsedEquation' not 'ParsedEquation'. 
Local variable name always starts with lower case letter